### PR TITLE
Media

### PR DESCRIPTION
**Pull request by JuliaKH**

### CODE REVIEW

File: src/sass/brands.sass [hunk 1]
```diff
@@ -1,22 +1,47 @@
...
+    border-width: 0 3px 3px 0
+    height: 20px
+    min-width: 20px
+    display: inline-block
+    padding: 3px
+    top: 60px
+  &__control--left:hover, &__control--right:hover
+    cursor: pointer
+  &__control--left
+    @include arwTransform(135deg)
```
Comment (LenaAviles) [inline src/sass/brands.sass]: you can use just `+` instead of `@include`

File: src/sass/mixins.sass [hunk 2]
```diff
@@ -1,3 +1,16 @@
 @mixin arwTransform($prop)
```
Comment (LenaAviles) [inline src/sass/mixins.sass]: what 'arw' means?

File: src/sass/mixins.sass [hunk 3]
```diff
@@ -1,3 +1,16 @@
...
   transform: rotate($prop)
   -webkit-transform: rotate($prop)
+
+@mixin brands-media-query($screen-width,$line-width)
```
Comment (LenaAviles) [inline src/sass/mixins.sass]: this is not good approach(we'll discuss it tomorrow)

File: src/sass/mixins.sass (hunk 2 above)
Comment (JuliaKH) [reply src/sass/mixins.sass]: arrow)

File: dist/style.css [hunk 4]
```diff
@@ -195,24 +195,44 @@
...
-  min-height: 400px; }
-
-.brands .flex-container__item {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  margin-top: 25px; }
+.brands {
+  position: relative; }
+  .brands .slider-block {
```
Comment (ArtMrsh) [inline dist/style.css]: add mix `brands__slider-block`, and than add position

File: dist/style.css [hunk 5]
```diff
@@ -236,39 +256,13 @@
...
-.slider__control_left {
-  transform: rotate(135deg);
-  -webkit-transform: rotate(135deg);
-  left: 20px; }
-
-.slider__control_right {
-  transform: rotate(-45deg);
-  -webkit-transform: rotate(-45deg);
-  right: 20px; }
+  .slider-block__wrapper ul, .slider-block__wrapper li {
```
Comment (ArtMrsh) [inline dist/style.css]: don't use selectors by tag

File: dist/style.css [hunk 6]
```diff
@@ -236,39 +256,13 @@
...
+    margin: 0;
+    padding: 0; }
 
 .brands .lines {
```
Comment (ArtMrsh) [inline dist/style.css]: `lines brands__lines`

File: index.html [hunk 7]
```diff
@@ -177,64 +177,64 @@ <h2>ARIA MAC-STD</h2>
...
-                <div id="prev-btn" class="slider__control slider__control_left"></div>
-                <div id="next-btn" class="slider__control slider__control_right"></div>
-            </div>
         </div>
     </div>
+    <div class="brands__control">
+        <div id="prev-btn" class="brands__control--left"></div>
+        <div id="next-btn" class="brands__control--right"></div>
+    </div>
     <div class="lines">
```
Comment (ArtMrsh) [inline index.html]: `lines brands__lines`

File: src/sass/brands.sass [hunk 8]
```diff
@@ -1,22 +1,47 @@
 @import "mixins"
+$desktop-width: 1287px
+$tablet-width: 800px
+$phone-width: 550px
 
 .brands
+  position: relative
   .slider-block
```
Comment (ArtMrsh) [inline src/sass/brands.sass]: `$__slider-block`

**Pull request closed without merging**